test/uk/gov/hmrc/apiplatformoutboundsoap/controllers/ConfirmationControllerSpec.scala

```diff
@@ -144,6 +163,14 @@ class ConfirmationControllerSpec extends AnyWordSpec with Matchers with GuiceOne
       verifyZeroInteractions(confirmationServiceMock)
     }
 
+    "handle an XML request with blank RelatesTo element" in new Setup {
```

Comment:
If you were feeling energetic you could make a placeholder in line 74 above and parameterise this test so that it ran maybe 4 versions with zero-length string, some spaces, tab character and newline character.